OasisDEX/oasis-borrow
# Ajna validation adjustment

# [Ajna validation adjustment](https://app.shortcut.com/oazo-apps/story/8318/ajna-validation-front-end-handling)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added handling to borrow validations
  
## How to test 🧪
  <Please explain how to test your changes>

- try to force all borrow validations and verify whether they are according to stories
